Doc/library/builtins.rst

```diff
@@ -9,7 +9,8 @@
 This module provides direct access to all 'built-in' identifiers of Python; for
 example, ``builtins.open`` is the full name for the built-in function
 :func:`open`.  See :ref:`built-in-funcs` and :ref:`built-in-consts` for
-documentation.
+documentation. For more information about built-in types and exceptions, See
```

Comment:
I think you can probably just combine these new links into the existing sentence to reduce redundancy.

Maybe something like this:

"For more information, see :ref:`built-in-funcs`, :ref:`built-in-consts`, :ref:`bltin-types` and :ref:`bltin-exceptions`"

Reply:
Thanks for the better solution, I think it's great.